py-polars/polars/config.py

```diff
@@ -1195,6 +1195,32 @@ def set_tbl_width_chars(cls, width: int | None) -> type[Config]:
         ----------
         width : int
             Maximum table width in characters.
+
+        Examples
+        --------
+        >>> df = pl.DataFrame(
+        ...     {
+        ...         "x": [1234567, -987654, 10101],
+        ...         "y": [1234.5, 100000.0, -7654321.25],
+        ...     }
+        ... )
+        >>> pl.Config.set_tbl_width_chars(16)  # doctest: +SKIP
+        # ...
```

Comment:
Could you put the before/after on a different line of code? Then we can actually run the examples and make sure they are correct.

Reply:
Thanks for the feedback.  I will try a better example that helps rather than hinders readability and put it on different lines so it can be checked with doctest.